administration/intro.adoc

```diff
@@ -173,7 +129,7 @@ No additional steps are required to extend OKD's web console for KubeVirt.
 
 The virtualization extension is automatically enabled when KubeVirt deployment is detected.
 
-Client side `virtctl` deployment
+Installing `virtctl` client tooling
```

Comment:
In the github preview these headings look strange:

```
Deploying on OKD ~~~~~~~~
```

```
Web user interface on OKD ^^^^^^^^^^^
```

```
Installing virtctl client tooling ~~~~~~~~~~~~
```

Probably because the length of the heading changed. Not sure if that would be rendered correctly.

Reply:
I had fixed for all these formatting issues in https://github.com/kubevirt/user-guide/pull/313, but it is obsolete now anyway after going back to markdown :man_shrugging: